source/appModules/lync.py

```diff
@@ -15,9 +15,13 @@ class NetUIRicherLabel(UIA):
 	def event_liveRegionChange(self):
 		# The base liveRegionChange event is not enough as Skype for Business concatinates recent chat messages from the same person within the same minute
 		# Therefore, specifically strip out the chat content and only report the most recent part added.
-		# The object's name contains the full message (I.e. person: content, timestamp)
-		# The object's value just contains the content
+		# The object's name contains the full message (I.e. person: content, timestamp) loosely separated by commas.
+		# Example string: "Michael Curran : , , Hello\nThis is a test , 10:45 am."
+		# Where person is "Michael Curran", content is "Hello\nThis is a test" and timestamp is "10:45 am" 
+		# The object's value just contains the content.
+		# Example: "Hello\nThis is a test"

```

Comment:
Maybe include in these examples the `\r\n` details. To show what the raw strings we get are like.